src/solvers/dg/containers_1d.jl

```diff
@@ -1,4 +1,113 @@
 
+# Container data structure (structure-of-arrays style) for DG elements
+# TODO: Taal refactor, remove u, u_t, u_tmp2, u_tmp3
+# TODO: Taal refactor, remove NVARS, POLYDEG?
+mutable struct ElementContainer1D{RealT<:Real, NVARS, POLYDEG} <: AbstractContainer
```

Comment:
Do I assume correctly that `src/solvers/dg/containers_[123]d.jl` just contains the contents (or some of them) of `src/solvers/dg/[123]d/containers.jl`, plus some methods that were previously defined in `src/solvers/dg/[123]d/dg.jl`, such as `init_elements` etc.?

Reply:
If not, please let me know what changed in addition such that I can give it a proper review.

Reply:
Your assumption is completely correct. The only real difference is the sorting: I've sorted everything by object (everything related to element containers, then interfaces, ...) instead of by verb (all all `count_*(`, then all `init_*(`, then all `init_*!(`, ...).